<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="HeaderPanel" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" stdset="0">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="BtnList">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Update Caster List</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="srctblbrows_qt.qrc">
              <normaloff>:/buttons/reload.bmp</normaloff>:/buttons/reload.bmp</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Address">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Caster Address:Port</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnUpdate">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Update Data Streams</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="srctblbrows_qt.qrc">
              <normaloff>:/buttons/load.bmp</normaloff>:/buttons/load.bmp</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TypeStr">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Data Streams</string>
            </property>
            <property name="text">
             <string>STR</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TypeCas">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>CAS</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TypeNet">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Networks</string>
            </property>
            <property name="text">
             <string>NET</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TypeSrc">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Source Table</string>
            </property>
            <property name="text">
             <string>SRC</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnMap">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>MAP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="FiltFmt">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RTCM 3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RTCM 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RAW</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="StaMask">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>STA</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnSta">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="Table1"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="Table3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="Table2"/>
       </item>
       <item>
        <widget class="QTableWidget" name="Table0"/>
       </item>
       <item>
        <widget class="QWidget" name="Panel2" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" stdset="0">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="Message">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="MenuOpen"/>
    <addaction name="MenuSave"/>
    <addaction name="separator"/>
    <addaction name="MenuQuit"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="MenuUpdateCaster"/>
    <addaction name="separator"/>
    <addaction name="MenuUpdateTable"/>
   </widget>
   <widget class="QMenu" name="View">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="MenuViewStr"/>
    <addaction name="MenuViewCas"/>
    <addaction name="MenuViewNet"/>
    <addaction name="separator"/>
    <addaction name="MenuViewSrc"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="MenuAbout"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="View"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="MenuOpen">
   <property name="text">
    <string>&amp;Open Source Table...</string>
   </property>
  </action>
  <action name="MenuSave">
   <property name="text">
    <string>&amp;Save Source Table...</string>
   </property>
  </action>
  <action name="MenuQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="MenuUpdateCaster">
   <property name="text">
    <string>Update &amp;Caster List</string>
   </property>
  </action>
  <action name="MenuUpdateTable">
   <property name="text">
    <string>Update Source &amp;Table</string>
   </property>
  </action>
  <action name="MenuViewStr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Stream List</string>
   </property>
  </action>
  <action name="MenuViewCas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Caster List</string>
   </property>
  </action>
  <action name="MenuViewNet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Network List</string>
   </property>
  </action>
  <action name="MenuViewSrc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;ource Table</string>
   </property>
  </action>
  <action name="MenuAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="srctblbrows_qt.qrc"/>
 </resources>
 <connections/>
</ui>
